allensdk/brain_observatory/nwb/__init__.py

```diff
@@ -406,17 +408,21 @@ def add_running_speed_to_nwbfile(nwbfile, running_speed,
 
 def add_stimulus_template(nwbfile: NWBFile,
                           stimulus_template: StimulusTemplate):
-    images = []
+    unwarped_images = []
+    warped_images = []
     image_names = []
     for image_name, image_data in stimulus_template.items():
         image_names.append(image_name)
-        images.append(image_data)
+        unwarped_images.append(image_data.unwarped)
+        warped_images.append(image_data.warped)
+
+    image_index = list(range(len(image_names)))
```

Comment:
Per @rly 's recommendation we should:

```
Set the timestamps of the ImageSeries to an array of nans of the same length as the data.
If you do this, I highly recommend adding a clear description of this referencing scheme
in the ImageSeries and your tables.
```

Reply:
@njmei I agree it's an improvement, but the existing behavior was to use the range index and it's unrelated to this ticket. I created a new ticket for that https://app.zenhub.com/workspaces/allensdk-10-5c17f74db59cfb36f158db8c/issues/alleninstitute/allensdk/1936

Reply:
Sounds good.